pmgr/pmgrAPI.py

```diff
@@ -0,0 +1,104 @@
+from .pmgrobj import pmgrobj
```

Comment:
I think this `pmgr/pmgrAPI.py` file is the most important one to add docstrings to, as it is the external API we'll use to link this into the hutch python tools.

I can figure out the intent of each of these by reading the code, but it would be extremely helpful to have docstrings like the following on all the public methods:
```
def get_config(self, pv):
    """
    Get the name of the configuration associated with a given PV base.

    Parameters
    ----------
    pv : str
        The pvname to use in the lookup.

    Returns
    -------
    config : str
        The name of the configuration associated with the PV.
    """
```
In that format we can also generate some auto html docs for the API page.

Reply:
I will see about documenting this today.  Then, time to figure out if pull requests can be amended, or if I need to make a new one. ;-)

Reply:
You can commit and push as you like (or amend, force push - but it's generally frowned upon) for it to be reflected here. Seems like you got that, though, but just to confirm 👍

Reply:
I figured I could just push to my fork, it's just a happy accident that this amends the pull request. ;-)

Reply:
Docstrings have been added to pmgrAPI and pmgrobj and a few other random places here and there.